README.md

```diff
@@ -1,35 +1,40 @@
 ![IDX header image](https://uploads-ssl.webflow.com/5ff39a496ca3e515d3359963/5ff48347c70c7d7c48fed7ea_image-idx-rethink-identity.png)
 
-# js-idx
+# IDX monorepo
 
-[![MIT license](https://img.shields.io/badge/License-MIT-blue.svg)](https://lbesson.mit-license.org/)
-[![](https://img.shields.io/badge/Chat%20on-Discord-orange.svg?style=flat)](https://chat.idx.xyz)
+[![Discord](https://img.shields.io/badge/Chat%20on-Discord-orange.svg?style=flat)](https://chat.idx.xyz)
 [![Twitter](https://img.shields.io/twitter/follow/identityindex?label=Follow&style=social)](https://twitter.com/identityindex)
 
-**js-idx** is a JavaScript/TypeScript implementation of the [IDX protocol](https://idx.xyz).
+Packages providing the JavaScript/TypeScript implementation of the [IDX protocol](https://idx.xyz).
+
 > ⚠️ IDX is in alpha. Libraries may be unstable and APIs are subject to change.
 
-## IDX: An identity protocol for open applications
+## Installation
+
+This monorepo uses workspaces from npm v7, included with Node v15.
+If you are using an older version of Node, make sure to install npm v7.
```

Comment:
I see, how can we make sure it still works with node 14? 
Also are we mixing lerna and npm workspaces? Can you clarify what we are doing with this monorepo setup?

Reply:
npm 7 works with node 14 and older, it's just not the version provided by default when installing node < 15.
Lerna is only used to run scripts in multiple packages, while the installation of the npm dependencies is done by npm itself.

Reply:
Ok, thanks for clarifying!